deblur/parallel_deblur.py

```diff
@@ -0,0 +1,71 @@
+# ----------------------------------------------------------------------------
+# Copyright (c) 2015, The Deblur Development Team.
+#
+# Distributed under the terms of the BSD 3-clause License.
```

Comment:
I thought everyone in QIIME already gave their permission to port everything in QIIME to be BSD, right? If that's the case, this should be fine, right?